app/src/client/mod.rs

```diff
@@ -456,7 +456,7 @@ fn GameView(hooks: &mut Hooks, show_debug: bool) -> Element {
                 }),
             }
             .el()
-            .with(docking(), ambient_layout::Docking::Bottom)
+            .with(docking(), ambient_layout::Docking::Top)
```

Comment:
This makes the debugger dropdown accessible as they don't go off the bottom of the screen